jctools-benchmarks/src/main/java/org/jctools/jmh/latency/QueueBurstCost.java

```diff
@@ -44,22 +44,25 @@
     @Param("true")
     boolean warmup;
     @Param(value = {"132000"})
-    String qCapacity;
-    Queue<AbstractEvent> q;
+    int qCapacity;
```

Comment:
qCapacity is a string to allow the <chunk,size> format conveniently from the command line.

Reply:
I have noted that one of the bench was hard-coding it: I can make both `String` if is better, ok?

Reply:
hard coding it?

Reply:
Yep: https://github.com/JCTools/JCTools/pull/273/files#diff-01928db23694a1a5eea730064785eb33L62

There was no means to change it in the original version: that's why I've tried to make both the bench to look similar

Reply:
I've addressed it in the last commit: let me resolve this